src/parser/common.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 WebAssembly Community Group participants
```

Comment:
```suggestion
 * Copyright 2023 WebAssembly Community Group participants
```

Reply:
ditto below

Reply:
It probably doesn't matter much, but I think for new files 2023 makes more sense. Though if it's just a rename, maybe either way.